frontend/src/pages/AccountsPage.jsx

```
@@ -48,169 +55,329 @@ export default function AccountsPage() {
-          className="mt-4 rounded-lg bg-red-600 px-4 py-2 text-sm text-white hover:bg-red-700"
+          onClick={() => refetch()}
+          className="mt-6 flex items-center gap-2 rounded-lg bg-red-600 px-5 py-2.5 text-sm font-medium text-white transition-all hover:bg-red-700 hover:shadow-lg"
         >
+          <RefreshCw className="h-4 w-4" />
           Try again
         </button>
       </div>
     );
   }
 
+  useEffect(
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)